tests/x11/x3270_ssl.pm

```diff
@@ -18,14 +18,28 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use power_action_utils 'power_action';
 
 sub run {
+
+    my ($self) = @_;
+
+    # Reboot the system when x11 launch fail to Workaround the Switch Console error
+    # (s390x limitation) s390x does not support snapshot to rollback the lastgood snapshot
+    # This workaround can avoid the blocked test fail from the last case
+    if (check_var('ARCH', 's390x')) {
```

Comment:
Will you put these conditions in one line? Or you leave it here for future using?

Reply:
Yes, it is more readable to put the conditions in one line, updated. Thanks.